org.caveman250.gtkeddit.json

```diff
@@ -0,0 +1,163 @@
+          "sha256": "6664e27c9a9cca81c29e35687f49f2e0d173a2fc9e98c3428311f707db532f8c"
+        }
+      ]
+    },
+    {
+      "name": "cairomm",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://cairographics.org/releases/cairomm-1.16.1.tar.xz",
+          "sha256": "6f6060d8e98dd4b8acfee2295fddbdd38cf487c07c26aad8d1a83bb9bff4a2c6"
+        }
+      ]
+    },
+    {
+      "name": "pangomm",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/pangomm/2.49/pangomm-2.49.1.tar.xz",
+          "sha256": "a2272883152618fddea016a62f50eb23b9b056ab3c08f3b64422591e6a507bd5"
+        }
+      ]
+    },
+    {
+      "name": "atkmm",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/atkmm/2.36/atkmm-2.36.1.tar.xz",
+          "sha256": "e11324bfed1b6e330a02db25cecc145dca03fb0dff47f0710c85e317687da458"
+        }
+      ]
+    },
+    {
+      "name": "gtkmm",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/gtkmm/4.2/gtkmm-4.2.0.tar.xz",
+          "sha256": "480c4c38f2e7ffcf58f56bb4b4d612f3f0cac9fd5908fd2cd8249fe10592a98b"
+        }
+      ]
+    },
+    {
+      "name": "jsoncpp",
+      "buildsystem": "meson",
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/open-source-parsers/jsoncpp",
+          "tag": "1.9.4"
+        }
+      ]
+    },
+    {
+      "name": "libmicrohttpd",
+      "sources":
+      [
+        {
+          "type": "archive",
+          "url": "https://ftp.gnu.org/gnu/libmicrohttpd/libmicrohttpd-0.9.73.tar.gz",
+          "sha256": "a37b2f1b88fd1bfe74109586be463a434d34e773530fc2a74364cfcf734c032e"
+        }
+      ]
+    },
+    {
+      "name": "boost",
+      "config-opts": [],
+      "cleanup": [ "/lib/libboost_*.a" ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/project/boost/boost/1.74.0/boost_1_74_0.tar.bz2",
+          "sha256": "83bfc1507731a0906e387fc28b7ef5417d591429e51e788417fe9ff025e116b1"
+        },
+        {
+          "type": "file",
+          "path": "boost-configure",
+          "dest-filename": "configure"
+        }
+      ]
+    },
+    {
+      "name": "websocketpp",
+      "buildsystem": "cmake",
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/zaphoyd/websocketpp",
+          "tag": "0.8.2"
+        }
+      ]
+    },
+    {
+      "name": "gtkeddit",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.com/caveman250/gtkeddit",
+          "tag": "0.1"
+        }
+      ],
+      "buildsystem": "cmake",
```

Comment:
```suggestion
      "buildsystem": "cmake-ninja",
```

Reply:
is there a reason this is preferable? i have always used cmake with gcc throughout development

Reply:
Ninja is an alternative to `make` that usually cuts off some time from the build time. It will use GCC for the build.

Reply:
fixed in latest.